testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/AcquireTokenFragment.java

```diff
@@ -43,6 +43,7 @@
 import android.widget.Toast;
 import android.widget.ToggleButton;
 
+import androidx.annotation.NonNull;
```

Comment:
```suggestion
```

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1793/commits/6362f0b11c7a210897be32dffc4f690506c2f349